README.md

```diff
@@ -31,7 +31,7 @@ A [RFC 5321](https://tools.ietf.org/html/rfc5321) compliant email.
 
 Arbitrary data encoded in JavaScript Object Notation. See https://www.json.org/.
 
-### [Mixed](src/Mixed.php)
+### [MixedScalar](src/MixedScalar.php)
```

Comment:
The GraphQL name of the type can still be `Mixed`, only the PHP class has to change.